src/backend/metal/src/command.rs

```diff
@@ -1927,6 +1927,15 @@ where
                 offset,
             );
         }
+        Cmd::InsertDebugMarker { ref name } => {
+            encoder.insert_debug_signpost(name);
+        }
+        Cmd::PushDebugMarker { ref name } => {
+            encoder.push_debug_group(name);
```

Comment:
There is one thing I wasn't sure about when thinking about this. Maybe you can clarify.
In Metal, is it valid to, say, push debug group inside a render pass, and pop it outside of the render pass?
In Vulkan, it seems totally valid.

Reply:
I believe it is but I'll double check with @warrenm, my spiritual advisor on Metal matters.